tests/CacheKeyNormalizerTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Cache\Tests;
+
+use PHPUnit\Framework\TestCase;
+use stdClass;
+use Yiisoft\Cache\Exception\InvalidArgumentException;
+use Yiisoft\Cache\CacheKeyNormalizer;
+
+use function fclose;
+use function fopen;
+use function json_encode;
+use function md5;
+
+final class CacheKeyNormalizerTest extends TestCase
+{
+    private CacheKeyNormalizer $normalizer;
+
+    public function setUp(): void
+    {
+        $this->normalizer = new CacheKeyNormalizer();
```

Comment:
Let's create it in a test case explicity

Reply:
I think it's OK as is.